rdrf/registry/groups/models.py

```diff
@@ -171,6 +174,10 @@ def is_carrier(self):
     def is_carer(self):
         return self.in_group(RDRF_GROUPS.CARER)
 
+    @property
+    def is_patient_or_delegate(self):
```

Comment:
Is there a better word than 'delegate', or word that means the thing _or_ their delegate / proxy

Reply:
caregiver?

Reply:
It sounds too close to 'carer' so could be misleading.

Reply:
I can't think of anything else. Maybe Yvette might have a good synonym or two up her sleeve?

Reply:
No luck, I'll just leave it as is for now